src/Symfony/Component/HttpKernel/HttpCache/Esi.php

```diff
@@ -84,7 +84,7 @@ public function hasSurrogateCapability(Request $request)
     public function addSurrogateCapability(Request $request)
     {
         $current = $request->headers->get('Surrogate-Capability');
-        $new = 'symfony2="ESI/1.0"';
+        $new = 'symfony3="ESI/1.0"';
```

Comment:
@hhamon 
Well, the specification says that the capability name [identifies a device token](https://www.w3.org/TR/edge-arch/), not an application or a server. 
'Device tokens must be unique within a request's Surrogate-Capabilities header'.  So, it is more likely an identifier inside a header.

Symfony processed this header in that way:
```php
return false !== strpos($value, 'ESI/1.0');
```
other application should probably process 'Surrogate-Capability' header in the similar way.

Do you have ideas where this change could be a BC?